lib/dialogs/buy/BuyDialog.dart

```diff
@@ -232,20 +228,13 @@ class _BuyDialogState extends State<BuyDialog> {
         buildWhen: (previous, current) => previous != current,
         builder: (context, state) {
           final bloc = context.read<BuyDialogBloc>();
-          final price = state.price.toStringAsFixed(6);
+          final price = state.aptBuyInfo.aptPrice.toStringAsFixed(6);
           final balance = state.balance;
-          final minReceived = state.minimumReceived.toStringAsFixed(6);
-          final priceImpact = state.priceImpact.toStringAsFixed(6);
-          final receiveAmount = state.receiveAmount.toStringAsFixed(6);
-          final totalFee = state.totalFee;
-          print("BuyDialog TokenAddress: ${state.tokenAddress}");
-          print("BuyDialog price: $price");
-          print("BuyDialog minReceived: ${state.minimumReceived}");
-          print("BuyDialog PriceImpact: ${state.priceImpact}");
-          print("BuyDialog ReceiveAmount: ${state.receiveAmount}");
-          print("BuyDialog Balance: ${state.balance}");
-          print("BuyDialog AxInput: ${state.axInputAmount}");
-          if (state.tokenAddress == null ||
+          final minReceived = state.aptBuyInfo.minimumReceived.toStringAsFixed(6);
```

Comment:
I think we should move this _toStringAsFixed_ business into the bloc because the bloc should produce a ready model for the view. The view shouldn't have to do further transformations on the model